doc/releases/changelog-dev.md

```diff
@@ -33,42 +33,116 @@
 
 <h4>Access an extended arsenal of quantum algorithms 🏹</h4>
 
-
-* The `FABLE` template is added for efficient block encoding of matrices. Users can now call FABLE to efficiently construct circuits according to a user-set approximation level. 
+* The Fast Approximate BLock-Encodings (FABLE) algorithm 
+  (https://arxiv.org/abs/2205.00081) is now accessible via the `qml.FABLE` 
+  template.
   [(#5107)](https://github.com/PennyLaneAI/pennylane/pull/5107)
 
-* Create the `qml.Reflection` operator, useful for amplitude amplification and its variants.
-  [(#5159)](https://github.com/PennyLaneAI/pennylane/pull/5159)
+  The usage of `qml.FABLE` is similar to `qml.BlockEncode` but provides a more
+  efficient circuit construction at the cost of a user-defined approximation 
+  level, `tol`. The number of wires that `qml.FABLE` operates on is `2*n + 1`, 
+  where `n` defines the dimension of the $2^n \times 2^n$ matrix that we want to 
+  block-encode.
 
   ```python
-  @qml.prod
-  def generator(wires):
-        qml.Hadamard(wires=wires)
-
-  U = generator(wires=0)
-
+  A = np.random.random((2, 2)) # n = 1, 2*n + 1 = 3
   dev = qml.device('default.qubit')
+
   @qml.qnode(dev)
   def circuit():
+      qml.FABLE(A, tol = 0.1, wires=range(3))
+      return qml.state()
+  ``` 
+
+  ```pycon
+  >>> circuit()
+  tensor([0.29906437+0.j, 0.20579772+0.j, 0.29906437+0.j, 0.20579772+0.j,
+          0.40070002+0.j, 0.45568333+0.j, 0.40070002+0.j, 0.45568333+0.j], requires_grad=True)
+  ```
```

Comment:
In the [docstrings example](https://xanaduai-pennylane--5574.com.readthedocs.build/en/5574/code/api/pennylane.FABLE.html) we give a fixed matrix and show it is successfully encoded. Shall we do the same here or do you think it makes it too complicated?

Reply:
The unitary matrix is a nice demonstration that you've encoded the input matrix correctly

Reply:
Want to come back to this one, I find the state much less useful in determing whether I've block encoded a matrix successfully. Instead, looking at the unitary matrix of the circuit should confirm this straight away.

Reply:
```python
A = np.array([[0.1, 0.2], [0.3, 0.4]])
dev = qml.device('default.qubit', wires=3)

@qml.qnode(dev)
def circuit():
    qml.FABLE(A, tol = 0.001, wires=range(3))  
    return qml.state()

mat = qml.matrix(circuit)()
qml.math.partial_trace(mat, indices=[1, 2])
```

```
tensor([[ 0.5       +0.j, -1.91150258+0.j],
        [ 1.91150258+0.j,  0.5       +0.j]], requires_grad=True)
```

🤔

Reply:
Not sure if `partial_trace` is expected here. You can do:
```python
>>> mat = qml.matrix(circuit)()
>>> 2 * mat[0:2, 0:2]
array([[0.1+0.j, 0.2+0.j],
       [0.3+0.j, 0.4+0.j]])
```
This is a simplification of the [docstrings example](https://xanaduai-pennylane--5574.com.readthedocs.build/en/5574/code/api/pennylane.FABLE.html).

Reply:
Ah okay this makes sense 😅